qui/tray/domains.py

```diff
@@ -506,32 +587,57 @@ def update_stats(self, vm, _event, **kwargs):
             kwargs['memory_kb'], kwargs['cpu_usage'])
 
     def initialize_menu(self):
-        for vm in self.qapp.domains:
-            if vm.is_running() and vm.klass != 'AdminVM':
-                self.add_domain_item(vm, None)
+        self.tray_menu.add(DomainMenuItem(None, self))
+        for vm in sorted(self.qapp.domains):
+            if vm.klass != 'AdminVM':
```

Comment:
I've just merged PR adding dom0 to the list, so this condition is not needed anymore.